sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -25,6 +25,7 @@
 import Queue
 import sys
 import threading
+from retrying import retry
```

Comment:
<!--new_thread; commit:eb2acfa855db99ab4b826344656ec29dcdb175e3; resolved:0-->
This is unused.